app/views/downloads/download_windows.html.erb

```diff
@@ -10,12 +10,28 @@
   <h1>Downloading Git</h1>
   <div class="callout downloading">
     <h3>Your download is starting...</h3>
-    <p>
-      <%= raw "You are downloading the latest (<strong>#{@download.version.name}</strong>) <strong>#{@bitness}</strong> version of <strong>Git for Windows</strong>. This is the most recent <a href='#{@project_url}'>maintained build</a>. It was released <strong>#{time_ago_in_words @download.release_date} ago</strong>, on #{@download.release_date.strftime("%Y-%m-%d")}." %>
-    </p>
-    <p>
-      <%= raw "<strong>If your download hasn't started, <a href=\"#{@download.url}\">click here to download manually</a>.</strong>" %>
-    </p>
+    <script>
```

Comment:
can we move this script into some .js file? there's a `app/assets/javascripts/site.js` with most of the javascript used in this app

Reply:
I think we can't because it is injecting HTML just in time and it is not possible if we move the script somewhere else.
But if you want to manipulate DOM – we can move it.